packages/backend/src/graphql/mutations/update-flow-config.ts

```diff
@@ -0,0 +1,34 @@
+import { type NotificationFrequency } from '@/models/flow'
+import Context from '@/types/express/context'
+
+type Params = {
+  input: {
+    id: string
+    notificationFrequency: NotificationFrequency
+  }
+}
+
+const updateFlowConfig = async (
+  _parent: unknown,
+  params: Params,
+  context: Context,
+) => {
+  let flow = await context.currentUser
+    .$relatedQuery('flows')
+    .findOne({
+      id: params.input.id,
+    })
+    .throwIfNotFound()
+
+  flow = await flow.$query().patchAndFetch({
+    config: {
+      errorConfig: {
+        notificationFrequency: params.input.notificationFrequency,
+      },
+    },
+  })
+
+  return flow
```

Comment:
nit: you can return the result of `patchAndFetch` directly instead of storing it in `flow`. Then you can use `const` for the `flow` var as well.

Reply:
i realised this is the case for `update-flow.ts` also, will update it, thanks!